change/@fluentui-react-native-e2e-testing-e9079191-951f-43ef-8361-8be50a28b7da.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "add additional tests for Text",
+  "packageName": "@fluentui-react-native/e2e-testing",
+  "email": "krsiler@microsoft.com",
+  "dependentChangeType": "patch"
```

Comment:
Is this supposed to be "patch" while the "type" is "minor"?

Reply:
hmmm I would think not but that's what `yarn change` resolved it to when I selected 'Minor` as the change type. I'm not sure how it makes that decision. Though I think this change should be a patch instead (since I'm not changing anything) so I'll update the change files.